api/team.js

```diff
@@ -157,6 +157,51 @@ router.post("/join/:code", async (req, res) => {
   });
 });
 
+// POST /api/team/join/:code
```

Comment:
Update comment to match this path and route description below
```suggestion
// POST /api/team/kick/:userid
```